data/json/recipes/nested.json

```
@@ -7100,6 +7100,30 @@
     ],
     "difficulty": 5
   },
+  {
+    "id": "nested_cardboard_armor",
+    "type": "nested_category",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_SUIT",
+    "name": "cardboard armor",
+    "description": "Recipes related to the construction of cardboard armor.",
```

```suggestion
    "description": "Recipes related to the construction of cardboard 'armor'.",
```
because the next description says 'weaponry'